engine/runtime/src/main/java/org/enso/interpreter/runtime/EnsoPolyglotJava.java

```diff
@@ -186,40 +212,10 @@ static void addToClassPath(
       EnsoContext ctx, Object whoIsIgnored, File path, boolean polyglotContextEntered)
       throws InteropException {
     var data = KEY.get(ctx);
-    data.hosted.addToClassPath(path, polyglotContextEntered);
-    data.guest.addToClassPath(path, polyglotContextEntered);
+    data.classPath.add(path);
```

Comment:
- just adding entries to `classPath`
- waiting for `findPolyglotJava()` to pick them up and `adjustClassPath()`
- delaying class path changes is OK - they aren't needed sooner anyway